example_project/example/models/example/http_events.sql

```diff
@@ -1,6 +1,14 @@
 /* Example following the quickstart guide at https://docs.decodable.co/docs/quickstart-guide */
-
-{{ config(watermark="`timestamp` AS `timestamp` - INTERVAL '0.001' SECOND") }}
+{{
+    config(
+        watermarks=[
+            {
+                "name": "timestamp",
+                "expression": "`timestamp` AS `timestamp` - INTERVAL '0.001'"
```

Comment:
```suggestion
                "expression": "`timestamp` AS `timestamp` - INTERVAL '0.001' SECOND"
```

I just had to add the unit (`SECOND`) to make this work! Our error message is a little funky, and threw me for a loop trying to find a bug in our adapter, so I'll see if we can improve that.

Reply:
awesome yeah this was totally an oversight ... it was getting late and I wasn't careful with the watermark refactor. did you test multiple watermarks? if not I can try to do that now

Reply:
ah, I'm getting this:

> Decodable: InvalidRequest: {'timestamp': '2024-12-13T17:51:36.457+00:00', 'message': 'Invalid stream. Reason: Invalid schema: Multiple watermark definitions are not supported yet.'}

would you prefer to let this error out here, or in the adapter code as argument validation? benefit of the latter is one less API call, but then we'd have to keep everything in sync

Reply:
I think we can let the API handle the validation. That's generally our preference, unless there's a really good reason to push some of it client-side, for the reason you mentioned of not having to keep all of our clients in sync with the business rules.

As for the error itself, we discussed this on Slack last week but this to repeat here this _is_ expected behavior. In practice you can only have 1 watermark, though the API takes an array.